src/main/java/com/nike/cerberus/operation/core/CreateDatabaseOperation.java

```diff
@@ -108,7 +108,8 @@ public boolean isRunnable(final CreateDatabaseCommand command) {
         try {
             cloudFormationService.getStackId(Stack.SECURITY_GROUPS.getFullName(environmentName));
         } catch (IllegalArgumentException iae) {
-            throw new IllegalStateException("The load balancer stack must exist to create the Route53 record!", iae);
+            logger.error("The load balancer stack must exist to create the Route53 record!");
```

Comment:
Is this error message right?  It looks like a copy/paste error?

Reply:
You would need to ask @sdford, i wasn't going to change it, just changing it from throwing an exception to logging and returning false

Reply:
It's a copy/paste error. There's no point in leaving it

Reply:
So what it supposed to be?

Reply:
It should be the SecurityGroup stack, at least that is what it is looking for in the try

Reply:
- [ ] Done.